### PR TITLE
GUI hw

### PR DESCRIPTION
**Pull request by SheyReiff**
Screenshot 2024-03-20 at 9 59 30 AM

### CODE REVIEW

File: src/main/java/reiff/fallingsand/SandComponent.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    public SandComponent(Sand sand) {
+        this.sand = sand;
+    }
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.setColor(Color.decode("#C2B280"));
```
Comment (andrewoid) [inline src/main/java/reiff/fallingsand/SandComponent.java]: make a constant for this color

Comment (SheyReiff) [reply src/main/java/reiff/fallingsand/SandComponent.java]: Should I make it local?

Comment (andrewoid) [reply src/main/java/reiff/fallingsand/SandComponent.java]: No.

File: src/main/java/reiff/fallingsand/SandComponent.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.setColor(standardSandColor);
+        for (int y = 0; y < sand.getHeight(); y++) {
+            for (int x = 0; x < sand.getWidth(); x++) {
+                if (sand.get(x, y) == 1) {
+                    g.fillRect(x, y, 3, 3);
```
Comment (andrewoid) [inline src/main/java/reiff/fallingsand/SandComponent.java]: This is going to draw a 3x3 rect which will be over other blank spaces.

Comment (SheyReiff) [reply src/main/java/reiff/fallingsand/SandComponent.java]: What size should be it be? 1, 1?

**Review by andrewoid — APPROVED** [review]

**Pull request merged**